### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**
Found via `codespell -L fo,reenable,renabled`

### CODE REVIEW

File: spec/support/helpers/environment_metdata_helper.rb
Comment (tombruijn) [inline spec/support/helpers/environment_metdata_helper.rb:1]: Can you undo this deletion?

Comment (kianmeng) [reply spec/support/helpers/environment_metdata_helper.rb:1]: @tombruijn Done.

**Comment by unflxw** [comment]
[`lib/appsignal/rack/body_wrapper.rb`](https://github.com/appsignal/appsignal-ruby/pull/1049/files#diff-542053b5df2402727a27d180e5f6f63bf8a646b81d72fa0c013de4177a6becaf) should not be in the diff.

**Comment by jeffkreeftmeijer** [comment]
This patch still removes https://github.com/appsignal/appsignal-ruby/pull/1049/files#diff-c242190a7b87c971e849da8f698666958a8a2e54ac5a96b0d27ad65d3d294be9. Could you revert that? Adding an extra commit to restore it is fine, we'll squash it when we merge.

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**